mission/src/components/ItemList/NavBar.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div class="nav-bar">
+    <!-- 컴포넌트화를 위한 방법 찾는 중입니다. -->
+    <!-- <NavItem :image="navigation.homeImage" :title="navigation.homeTitle" /> -->
+    <div class="home-nav">
+      <img data-test="home-image" src="@/assets/images/home.svg" />
+      <div data-test="home-title">{{ navigation.homeTitle }}</div>
+    </div>
+    <div class="like-nav">
+      <img data-test="like-image" src="@/assets/images/like.svg" />
+      <div data-test="like-title">{{ navigation.likeTitle }}</div>
+    </div>
+    <div class="cart-nav">
+      <img data-test="cart-image" src="@/assets/images/cart.svg" />
+      <div data-test="cart-title">{{ navigation.cartTitle }}</div>
+    </div>
+    <div class="mypage-nav">
+      <img data-test="mypage-image" src="@/assets/images/myPage.svg" />
+      <div data-test="mypage-title">{{ navigation.myPageTitle }}</div>
+    </div>
+  </div>
+</template>
+
+<script>
+// import NavItem from './NavItem.vue';
+
+export default {
+  name: 'NavBar',
+  // components: { NavItem },
+  data() {
+    return {
+      navigation: {
+        homeImage: '@/assets/images/home.svg',
+        homeTitle: '홈',
+        likeImage: '@/assets/images/like.svg',
+        likeTitle: '찜',
+        cartImage: '@/assets/images/cart.svg',
+        cartTitle: '장바구니',
+        myPageImage: '@/assets/images/mypage.svg',
+        myPageTitle: '마이 페이지',
+      },
```

Comment:
```suggestion
      navigation: [{
            image: '@/assets/images/home.svg',
            title: '홈'
      }, {
            image: '@/assets/images/like.svg',
            title: '찜'
      }, ...]
```
메뉴를 배열로 구성 하여 v-for 통해 화면에 보여준다면, 
추후 메뉴가 추가 되는 경우에도 간단하게 추가 할 수 있습니다 :D